test/testdata/deployednettemplates/recipes/devnet/gen_topology.py

```diff
@@ -0,0 +1,31 @@
+node_types = {"RN":5 }
+node_size = {"RN":"-m5d.4xl"}
+regions = [
+    "AWS-US-EAST-1",
+    "AWS-US-WEST-2",
+    "AWS-SA-EAST-1",
+    "AWS-EU-NORTH-1",
+    "AWS_AP_SOUTHEAST-2"
+]
+network="DevNet"
+f = open("topology.json", "w")
+f.write("{ \"Hosts\":\n  [")
+
+region_count = len(regions)
+first = True
+for  x in node_types:
+    node_type = x
+    node_count = node_types[x]
+    region_size = node_size[x]
+    for i in range(node_count):
+        node_name = node_type + str(i+1) + "-" + network
+        region = regions[i%region_count]
+        if (first ):
+            first = False
+        else:
+            f.write(",")
+        f.write ("\n    {\n      \"Name\": \"" + node_name + "\",\n      \"Template\": \"" + region + region_size + "\"\n    }"  )
```

Comment:
I would look at the `json` stdlib module so you're not having to write json by hand.

Reply:
Updated to use json.dumps().